src/victory-util/transitions.js

```diff
@@ -175,51 +185,60 @@ function getChildPropsOnEnter(animate, data, enteringNodes) {
  * For each transition type (enter, exit, move), find the longest duration
  * of each type from any of the children.
  *
- * @param  {Array}  children            `this.props.children` from parent component.
+ * @param  {Object} props            `this.props` from parent component.
  * @param  {Object} childrenTransitions Child transitions data, as calculated by the
  *                                      `getInitialTransitionState` function.
- * @param  {Object} parentAnimate       `this.props.animate` from parent component, to
- *                                      be used for transition duration defaults.
  *
  * @return {Object}                     `{ exit, enter, move }`
  */
-function getTransitionDurations(children, childrenTransitions, parentAnimate) {
+function getTransitionDurations(props, childrenTransitions) {
   if (!childrenTransitions) {
     return {};
   }
-
-  return children.reduce((durations, child, idx) => {
-    if (
-      childrenTransitions[idx] &&
-      childrenTransitions[idx].exiting &&
-      child.props.animate &&
-      child.props.animate.onExit &&
-      child.props.animate.onExit.duration > durations.exit
-    ) {
-      durations.exit = child.props.animate.onExit.duration;
-    }
-    if (
-      childrenTransitions[idx] &&
-      childrenTransitions[idx].entering &&
-      child.props.animate &&
-      child.props.animate.onEnter &&
-      child.props.animate.onEnter.duration > durations.enter
-    ) {
-      durations.enter = child.props.animate.onEnter.duration;
-    }
-    if (
-      child.props.animate &&
-      child.props.animate.duration > durations.move
-    ) {
-      durations.move = child.props.animate.duration;
-    }
-
-    return durations;
-  }, {
+  const parentAnimate = props.animate || {};
+  const parentDurations = {
     exit: parentAnimate.onExit && parentAnimate.onExit.duration || null,
     enter: parentAnimate.onEnter && parentAnimate.onEnter.duration || null,
     move: parentAnimate.duration || null
-  });
+  };
+
+  const children = React.Children.toArray(props.children);
+  if (children) {
```

Comment:
The docs don't say that `React.Children.toArray` will return a falsey value if `children` are falsey.  Is that the case?